include/attribute-based/bsw/ciphertext-policy/BSWCiphertextPolicyAttributeBasedEncryption.h

```diff
@@ -0,0 +1,33 @@
+#ifndef __CRYPTID_CRYPTID_ABE_H
```

Comment:
This feature guard is not following the structure of the library, check [BonehFranklinIdentityBasedEncryption.h](https://github.com/cryptid-org/cryptid-native/blob/master/include/identity-based/encryption/boneh-franklin/BonehFranklinIdentityBasedEncryption.h) for the correct usage. Also check if this issue occurs in any other files.